ydb/public/lib/ydb_cli/commands/ydb_profile.cpp

```diff
@@ -1058,7 +1059,15 @@ void TCommandReplaceProfile::Parse(TConfig& config) {
 
 int TCommandReplaceProfile::Run(TConfig& config) {
     auto profileManager = CreateProfileManager(config.ProfileFile);
-    profileManager->RemoveProfile(ProfileName);
+    if (profileManager->HasProfile(ProfileName)) {
+        if (!Force) {
+            Cout << "Profile data will be permanently removed. Continue? (y/n): ";
```

Comment:
`Current profile will be replaced with a new one. All current profile data will be lost. Continue?`

Otherwise its sounds too dangerous